apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-block-editor-close-click.js

```diff
@@ -10,7 +10,8 @@ import tracksRecordEvent from './track-record-event';
  */
 export default () => ( {
 	id: 'wpcom-block-editor-close-click',
-	selector: '.edit-post-header .edit-post-fullscreen-mode-close',
+	selector:
+		'.edit-post-header .edit-post-fullscreen-mode-close:not(.wpcom-block-editor-nav-sidebar-toggle-sidebar-button__button), .wpcom-block-editor-nav-sidebar-nav-sidebar__home-button',
 	type: 'click',
```

Comment:
Im starting to second guess our decision on this now.  Looking further there is also a `calypso_editor_sidebar_editor_close` set up:

https://github.com/Automattic/wp-calypso/blob/f2af21760128db3bdc9fbf287b09f459c2cb0b8d/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx#L131

Although, I am not seeing this one firing when I use this close button for some reason. 🤔   I see it in recent tracks history though, there must be some inconsistency with the conditional, maybe it needs to record the event regardless of that condition.